ios/Capacitor/Capacitor/CapacitorBridge.swift

```diff
@@ -444,26 +427,27 @@ internal class CapacitorBridge: NSObject, CAPBridgeProtocol {
      */
     // swiftlint:disable:next function_body_length
     func handleJSCall(call: JSCall) {
-        guard let plugin = self.plugin(withName: call.pluginId) ?? self.loadPlugin(pluginName: call.pluginId) else {
-            CAPLog.print("⚡️  Error loading plugin \(call.pluginId) for call. Check that the pluginId is correct")
-            return
+        let load = {
```

Comment:
Not really sure if this is wholly neccessary, I mostly tried to keep the behavior the same, but I'm not sure when a plugin would be called before the bridge has initialized all the plugins 🤷‍♂️. In all my debugging of this `handleJSCall(call:)` method, I've never gotten it to coalesce to the load method.

Reply:
Huh, yeah, that's interesting - I'd love if we had some real world code we could run regressions on like Swift.org does.